src/popup/components/TransactionItem.vue

```diff
@@ -84,16 +87,19 @@ export default defineComponent({
     const getAmountFiat = useGetter('getAmountFiat');
     const activeNetwork = useGetter('activeNetwork');
     const account = useGetter('account');
+    const getTxDirection = useGetter('getTxDirection');
+    const availableTokens = useState('fungibleTokens', 'availableTokens');
```

Comment:
in other files we had useState above useGetter, and normaly we had empty line between diffrent composables